scripts/ci/check_legacy_growth_guard.py

```
@@ -10756,6 +10768,1241 @@ def _parses_environment_directly(tree: ast.Module) -> bool:
+        "dotenv.load_dotenv",
+        "get_runtime_env_name",
+        "logging.basicConfig",
+        "metadata.to_fastapi_kwargs",
+        "os.getenv",
+    }
+)
+
+
+def _canonical_application_has_closed_call_grammar(tree: ast.Module) -> bool:
+    return all(
+        (name := _exact_dotted_name(node.func)) in _CANONICAL_APPLICATION_CALLS
```

Flake8 (7.3.0)

[error] local variable 'name' is assigned to but never used

(F841)